articles/active-directory/develop/quickstart-v2-java-webapp.md

```diff
@@ -34,52 +34,66 @@ When you've completed the guide, your application will accept sign-ins of person
 
 To run this sample you will need:
 - [Java Development Kit (JDK)](https://openjdk.java.net/) 8 or greater, and [Maven](https://maven.apache.org/).
+- An Internet connection
+- An Azure Active Directory (Azure AD) tenant. For more information on how to get an Azure AD tenant, see [How to get an Azure
+AD tenant](https://azure.microsoft.com/en-us/documentation/articles/active-directory-howto-tenant/) 
+- A user account in your Azure AD tenant.
```

Comment:
When I originally wrote the quickstart it had these requirements, but feedback was that these things were obvious and therefore we should get rid of them.

Reply:
Oh atleast regarding Azure AD tenant I think we should mention right ?

Reply:
Agree that we keep the Azure AD tenant point but we can get rid of the internet connection and the user account points.

Reply:
Updated.